modules/tensor_mechanics/test/tests/dynamics/wave_1D/tests

```diff
@@ -34,6 +34,19 @@
                    " in a linear elastic material with both Rayleigh damping and"
                    " numerical damping resulting from Hilber-Hughes-Taylor (HHT) time integration."
   [../]
+  [./rayleigh_hht_ad]
+    type = 'Exodiff'
```

Comment:
When doing automatic differentiation work. I generally require that a `PetscJacobianTester` test be added

Reply:
I have added a PetscJacobianTester test.